packages/nimble-components/src/dialog/tests/dialog.stories.ts

```diff
@@ -49,9 +56,33 @@ const content = {
     [ExampleContentType.longContent]: longContent
 } as const;
 
+const sizeDescription = `
+Size of a nimble dialog.
+
+See the Sizing section below for information on controlling the size of the dialog.
```

Comment:
When on the canvas page you can't see the usage docs so this could be confusing

```suggestion
See the Sizing section of the Usage Docs for information on controlling the size of the dialog.
```